packages/builder/src/schemas.zod.ts

```diff
@@ -457,3 +457,10 @@ export const chainDefinitionSchema = z
       })
       .deepPartial()
   );
+
+export const routerSchema = z.object({
```

Comment:
oh wow I just realized we arent just installing the router by default; we are embedding it into the builder? I guess thats fine, it is a super powerful action that probably deserves a standard of its own